api/types.go

```diff
@@ -0,0 +1,47 @@
+package api
+
+type Context struct {
+	State DebuggerState `json:"state"`
```

Comment:
That works for me - sorry to nitpick on naming - especially as I've not had
as much time to work on delve as I'd hoped.

On 27 March 2015 at 15:39, Dan Mace <notifications@github.com> wrote:

> In api/types.go
> <https://github.com/derekparker/delve/pull/88#discussion_r27324982>:
>
> > @@ -0,0 +1,47 @@
> > +package api
> > +
> > +type Context struct {
> > +	State DebuggerState `json:"state"`
>
>  I'd just like to point out that context is pecifically used for request
> scoping in Go
> https://blog.golang.org/context
> Potentially a more specific term could be used
>
> How about DebuggerContext as @pmorie <https://github.com/pmorie>
> originally mentioned?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/derekparker/delve/pull/88/files#r27324982>.
>